src/System.Private.ServiceModel/tests/Common/Scenarios/ScenarioTestTypes.cs

```diff
@@ -793,6 +797,30 @@ public Guid CallbackGuid
         }
     }
 
+    public XmlCompositeType XmlCallbackGuid
+    {
+        get
+        {
+            if (_xml_tcs.Task.Wait(ScenarioTestHelpers.TestTimeout))
+            {
+                return _xml_tcs.Task.Result;
+            }
+            throw new TimeoutException(string.Format("Not completed within the alloted time of {0}", ScenarioTestHelpers.TestTimeout));
+        }
+    }
+
+    public ComplexCompositeType DataContractCallbackGuid
+    {
```

Comment:
Possibly add a comment that no service contract is permitted to use this type.  It is reserved for the scenario that is testing that its appearance in a callback contract generates a serializer.  Probably same for the one above.